src/variorum/variorum.h

```diff
@@ -74,6 +74,14 @@ int variorum_cap_each_core_frequency(int cpu_freq_mhz);
 ///// @return Error code.
 //int set_each_gpu_power_limit(int gpu_power_limit);
 
+/// @brief Set the frequency of the target processor.
+///
+/// @param [in] socketid Target socket ID.
+/// @param [in] socket_freq_mhz Desired socket frequency in MHz.
+///
+///// @return Error code.
+int variorum_set_socket_frequency(int socketid, int socket_freq_mhz);
```

Comment:
Posting a question that Tapasya brought up yesterday: When you're setting `socketid`, is this switching between the big and little core or does this API set the same frequency to both big and little?

Reply:
I think having the "socketid" in the API means that we pick big and little based on the ID and specify the value. What I was confused about was whether there is a ratio that needs to be maintained in hardware. I am not too familiar with ARM, so my question may be rather dumb. As in, what if someone sets the big frequency to be lower than the little frequency? Is that a problem? Is there a ratio that needs to be maintained between the two to ensure safe operation of the chip?

Reply:
I think if `socketid` indicates big or little, it may make sense to change the variable name and comment to better reflect this. When I think socket ID, I think socket 0, socket 1, ... in a homogeneous sense.

Reply:
Although not that I'm thinking about it, if this is applied to other architectures, the big/little variable name won't make sense.

Reply:
Fix the documentation.

Reply:
Also remove extra slashes.